tools/maint/create_release.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+# Copyright 2022 The Emscripten Authors.  All rights reserved.
+# Emscripten is available under two separate licenses, the MIT license and the
+# University of Illinois/NCSA Open Source License.  Both these licenses can be
+# found in the LICENSE file.
+
+from datetime import datetime
+import os
+import subprocess
+import sys
+
+script_dir = os.path.dirname(os.path.abspath(__file__))
+root_dir = os.path.dirname(os.path.dirname(script_dir))
+
+sys.path.append(root_dir)
+from tools import shared, utils
+
+
+def main():
+  if subprocess.check_output(['git', 'status', '-uno', '--porcelain'], cwd=root_dir).strip():
+    print('tree is not clean')
+    return 1
+
+  shared.set_version_globals()
+
+  old_version = [shared.EMSCRIPTEN_VERSION_MAJOR, shared.EMSCRIPTEN_VERSION_MINOR,
+                 shared.EMSCRIPTEN_VERSION_TINY]
+  new_version = list(old_version)
+  new_version[2] += 1
+
+  old_version = '.'.join(str(v) for v in old_version)
+  new_version = '.'.join(str(v) for v in new_version)
+
+  print('Creating new release: %s' % new_version)
+
+  version_file = os.path.join(root_dir, 'emscripten-version.txt')
+  changelog_file = os.path.join(root_dir, 'ChangeLog.md')
+
+  old_content = utils.read_file(version_file)
+  utils.write_file(version_file, old_content.replace(old_version, new_version))
+
+  changelog = utils.read_file(changelog_file)
+  marker = f'{old_version} (in development)'
+  pos = changelog.find(marker)
+  assert pos != -1
+  pos += 2 * len(marker) + 1
+
+  # Add new entry
+  today = datetime.now().strftime('%m/%d/%y')
+  new_entry = f'{old_version} - {today}'
+  new_entry = '\n\n' + new_entry + '\n' + ('-' * len(new_entry))
+  changelog = changelog[:pos] + new_entry + changelog[pos:]
+
+  # Update the "in development" entry
+  changelog = changelog.replace(f'{old_version} (in development)', f'{new_version} (in development)')
+
+  utils.write_file(changelog_file, changelog)
+
+  branch_name = 'version_' + new_version
```

Comment:
I think the branch name and commit message would make more sense using old_version here instead of new_version. old_version is the version we are actually releasing with these changes. the "Mark {old_version} as released" commit messages we've been using make more sense than this IMO (especially since we just created a PR called {old_version} in emsdk